assets/src/edit-story/components/header/buttons/preview.js

```diff
@@ -118,11 +126,25 @@ function Preview() {
     []
   );
 
+  const label = __('Preview', 'web-stories');
   return (
     <>
-      <Outline onClick={openPreviewLink} isDisabled={isSaving || isUploading}>
-        {__('Preview', 'web-stories')}
-      </Outline>
+      <Tooltip
+        title={label}
+        placement={TOOLTIP_PLACEMENT.BOTTOM}
+        hasTail={true}
```

Comment:
```suggestion
        hasTail
```

I actually thought we had an eslint rule for this?

@swissspidy, could we add [`react/jsx-boolean-value`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md) (config'ed to `never`) if we haven't already?

Reply:
Sure. That rule is not in the recommended ruleset, so needs to be manually added.